packages/core/playwright.config.ts

```diff
@@ -7,13 +7,14 @@ const config: PlaywrightTestConfig = {
   testMatch: '*.e2e.ts',
   use: {
     baseURL: 'http://localhost:3333',
+    browserName: 'chromium',
   },
   webServer: {
     command: 'serve -p 3333',
     port: 3333,
     reuseExistingServer: !process.env.CI,
   },
-  workers: 4,
+  workers: 8,
```

Comment:
```suggestion
  workers: 8,
  reporter: 'line',
  projects: [
    {
      name: 'Chromium',
      use: { browserName: 'chromium' },
    },
    {
      name: 'Firefox',
      use: { browserName: 'firefox' },
    },
    {
      name: 'WebKit',
      use: { browserName: 'webkit' },
    },
  ],
```

Reply:
In Playwright you just specify 'Firefox' I believe.

Reply:
Lets give it a shot in afternoon, I want to see how much time it will take to run all the tests if we have all the browsers :)

Reply:
@mJarsater & @timrombergjakobsson here is the outcome:

<img width="834" alt="Screenshot 2024-03-28 at 15 20 51" src="https://github.com/scania-digital-design-system/tegel/assets/17004063/913e9255-a682-478e-91e1-373facd6e7ff">

Running test on local took over 2min. Every browser generated its own screenshot. I think we need to change approach to testing only component changed on commit and leave entire system check once PR is submitted.